config/_default/menus/menus.en.yaml

```diff
@@ -2455,16 +2455,21 @@ main:
     identifier: observability_pipelines
     parent: log_management_heading
     weight: 175000
+  - name: Quickstart
+    url: observability_pipelines/quickstart/
+    parent: observability_pipelines
+    identifier: observability_pipelines_quickstart
+    weight: 1
```

Comment:
Just to reiterate what's been raised in discussions:

1. Having both a "Quickstart" and "Installation" page is confusing since they are often used as the same thing.
2. While this is a "quickstart" for users who have deployed the Datadog Agent, it is not for users who have deployed Splunk, Otel, or multiple tools. This is why I originally wanted have a generic "install" page that led into individual guides for integration (Datadog Agent, Splunk Forwarders, etc). The reality is that most companies are going to have multiple tools deployed and will want to integrate them one at a time.

Finally, I realize 2 can happen after we introduce more guides like this, but I wanted to mention it since it has been a main them of my feedback. Eventually I think we'll forced to make this separation and offer integration specific guides.

Reply:
I've moved this guide to the Getting Started section and also linked to it from the Installation page as we had discussed in our mtg yesterday.